spec/spec_helper.rb

```diff
@@ -185,36 +187,6 @@ def processing?
   end
 end
 
-# Turn on elastic indexing for certain models. We do this selectively b/c
```

Comment:
Moved these two methods to `spec/support/es_toggling.rb` to place coupled piece of functionality in a separate place.